Cargo.toml

```diff
@@ -29,14 +29,14 @@ commit_verify = "0.11.0-beta.6"
 bp-core = "0.11.0-beta.6"
 bp-seals = "0.11.0-beta.6"
 bp-std = "0.11.0-beta.6"
-bp-electrum = "0.11.0-beta.6"
-bp-esplora = { version = "0.11.0-beta.6", default-features = false, features = ["blocking"] }
```

Comment:
Why do we re-order crates here?

Reply:
Thank you for the reminder. I will restore the order of the crates shortly.

During development, I centralized the crates that needed to be referenced via git, which led to changes in the order.